src/js/utils/autoComma.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-param-reassign */
+function comma(str) {
+  str = String(str);
+  return str.replace(/(\d)(?=(?:\d{3})+(?!\d))/g, '$1,');
```

Comment:
```suggestion
  return String(str).replace(/(\d)(?=(?:\d{3})+(?!\d))/g, '$1,');
```
파라미터에 재할당 하지 않고 바로 리턴할 수 있지 않을까요? 🙂

Reply:
맞네요. 간단한 방법이 있었는데 놓쳤습니다. 알려주셔서 감사해요! 👍

Reply:
콤마 찍는 방법으로 [toLocaleString](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/toLocaleString) Array 메소드도 있으니 참고해주세요~